resources/sass/_variables.scss

```diff
@@ -7,6 +7,9 @@ $font-size-base: 0.9rem;
 $line-height-base: 1.6;
 
 // Colors
+$white: #fff;
+$black: #000;
```

Comment:
Why do you define black and white when you don't use them yourselves? :D

Reply:
Otherwise this caused a bug in the SASS-compiler. This is a remnant of #243. :D